bin/heatmapper

```diff
@@ -511,8 +555,8 @@ def main(args):
     if args.outFileNameMatrix:
         hm.saveMatrixValues(args.outFileNameMatrix)
 
-    if args.outFileNameData:
-        hm.saveTabulatedValues(args.outFileNameData)
+    #if args.outFileNameData:
```

Comment:
The writer function was never written. This should be on the to do list for the next release, though.